deny.toml

```diff
@@ -106,6 +106,7 @@ allow = [
     "MIT-0",
     "Apache-2.0",
     "BSD-3-Clause",
+    "BSD-2-Clause",
```

Comment:
Should we have a comment right in this PR explaining this addition?
(not inside source code)

Reply:
I figured this license is innocuous enough to not need it.  When I added the license check, I was more concerned about catching us adding code that used a questionable license.

If you still want it, I can add it.  This was for blake3 and both can be removed in the future.